versions/1/integration/elasticsearch/transform/manifest.spec.yml

```
@@ -0,0 +1,14 @@
+##
+## Describes the specification for a transform's manifest.yml file
+##
+spec:
+  type: object
+  additionalProperties: false
+  definitions:
+    index_template:
+      $ref: "../../data_stream/manifest.spec.yml#/properties/elasticsearch/properties/index_template"
+  properties:
+    dest_index_template:
```

Nit. No need to use abbreviations.
```suggestion
    destination_index_template:
```